server-script/pom.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0"?>
 <!-- 
-/* 
- * The contents of this file are subject to the Terracotta Public License Version
- * 2.0 (the "License"); You may not use this file except in compliance with the
- * License. You may obtain a copy of the License at 
+/**
```

Comment:
This file has _odd_ comment content.  If it passes license validation, keep it but it's strange.